### PR TITLE
Test references non existing request fields

### PR DESCRIPTION
**Pull request by th-ei**
## Description

```
Test references non existing request fields
```
## Motivation and Context

spontaneous fixes
## How Has This Been Tested?

reran chain-tests.js
## Checklist:

- [x] I have added a [Signed-off-by](https://github.com/hyperledger/fabric/blob/master/CONTRIBUTING.md#legal-stuff)
- [x] Either no new documentation is required by this change, OR I added new documentation
- [x] Either no new tests are required by this change, OR I added new tests
- [-] I have run [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports), [go vet](https://golang.org/cmd/vet/), and [golint](https://github.com/golang/lint). I have cleaned up all valid errors and warnings in code I have added or modified. These tools may generate false positives. Don't be worried about ignoring some errors or warnings. The goal is clean, consistent, and readable code.

Signed-off-by: Thomas Eirich eir@zurich.ibm.com

### CODE REVIEW

File: sdk/node/src/hlc.ts [hunk 1]
```diff
@@ -1466,11 +1466,11 @@ export class Peer {
      * This is a temporary hack until event notification is implemented.
      * TODO: implement this appropriately.
      */
-    private waitToComplete(eventEmitter:events.EventEmitter): void {
+    private waitToComplete(eventEmitter:events.EventEmitter, rmsg:Buffer): void {
         debug("waiting 5 seconds before emitting complete event");
         var emitComplete = function() {
             debug("emitting completion event");
-            eventEmitter.emit('complete');
+            eventEmitter.emit('complete',rmsg);
```
Comment (corecode) [inline sdk/node/src/hlc.ts:1473]: space after comma?

**Comment by christo4ferris** [comment]
@th-ei can you please update the PR description to elaborate on the changes you made and the motivation behind them? "spontaneous fixes" is wholly inadequate.

**Comment by ghaskins** [comment]
@th-ei thank you for the patch.  In addition to the comments above, please use rebase instead of merge.

File: sdk/node/src/hlc.ts (hunk 1 above)
Comment (srderson) [inline sdk/node/src/hlc.ts:1473]: @th-ei please run `goimports`

Comment (srderson) [inline sdk/node/src/hlc.ts:1473]: ha, nm. Realized this is a TypeScript file

**Comment by srderson** [comment]
PR currently has a merge conflict and must be rebased. @smithbk @angrbrd any comments?

**Comment by christo4ferris** [comment]
reverify Jenkins

**Comment by th-ei** [comment]
This is obsolete by now

**Pull request closed without merging**